File: dotnet/Domain/Models/Order.cs
namespace Domain.Models
{
    public class Order
    {
        public int Id { get; set; }
        public decimal Price { get; set; }
        public int Quantity { get; set; }
        public required string UserName { get; set; }
        public int StockId { get; set; }
        public required string StockName { get; set; }
    }
}

File: angular/src/app/order/order.ts
export class Order {
  id: number;
  userName: string;
  price: number;
  quantity: number;
  stockId: number;
  stockName: string;

  constructor(order: Order) {
    this.id = order.id;
    this.userName = order.userName;
    this.price = order.price;
    this.quantity = order.quantity;
    this.stockId = order.stockId;
    this.stockName = order.stockName;
  }
}
